### PR TITLE
f-fix/gh-1366-on-team-calendar-icon-for-holidays-missing

### PR DESCRIPTION
**Pull request by popovicbojan**
Resolves: #1366

### CODE REVIEW

File: src/main/frontend/src/pages/team-calendar/components/styled.js [hunk 1]
```diff
@@ -249,3 +249,24 @@ export const ViewIcon = styled(({ isSelected, ...rest }) => (
     }
   }
 `;
+
+export const HolidayIconContainer = styled.div`
+  float: right;
```
Comment (aberonja) [inline src/main/frontend/src/pages/team-calendar/components/styled.js:254]: Lets not use float if we use flex. Try to do it without it.

Comment (aberonja) [reply src/main/frontend/src/pages/team-calendar/components/styled.js:254]: \n

File: src/main/frontend/src/pages/team-calendar/components/styled.js [hunk 2]
```diff
@@ -249,3 +249,24 @@ export const ViewIcon = styled(({ isSelected, ...rest }) => (
...
+  @media (max-width: 767px) {
+    position: relative;
+    left: 17px;
+    bottom: 30px;
+    float: none;
```
Comment (aberonja) [inline src/main/frontend/src/pages/team-calendar/components/styled.js:259]: Same here.

File: src/main/frontend/src/pages/team-calendar/components/styled.js [hunk 3]
```diff
@@ -249,3 +249,24 @@ export const ViewIcon = styled(({ isSelected, ...rest }) => (
...
+
+export const DayNumberContainer = styled.div`
+  display: inline-block;
+  padding: 4px 8px;
+  z-index: 1;
+`;
+
+export const CalendarViewImage = styled.img`
+  width: 20px;
+  margin-right: 1px;
```
Comment (aberonja) [inline src/main/frontend/src/pages/team-calendar/components/styled.js:271]: This is a bit odd, why do we have this?

Comment (popovicbojan) [reply src/main/frontend/src/pages/team-calendar/components/styled.js:271]: This is styled image for holiday icon in Team Calendar tab calendar view. I think it looks a little bit nicer with margin 1px than without it. :)
image

**Pull request merged**